monitoring/node-monitoring.yml

```diff
@@ -27,7 +27,7 @@ services:
     environment:
       METRICS_EXPORTER_BASIC_AUTH: "${BASIC_AUTH_USER_CREDENTIALS}"
       METRICS_EXPORTER_ROOT_API_URL: "${NODE_END_POINT}"
-      METRICS_EXPORTER_LEDGER_FOLDER: /data
+      #METRICS_EXPORTER_LEDGER_FOLDER: /data
```

Comment:
is this commented out on purpose?

Reply:
Yeah.. as that was throwing exception when user doesn't setup /data directory. Also the dashboard which we sharing is not picking up any info.. I will remove the line itself